PyToQlik Documentation.md

```diff
@@ -0,0 +1,142 @@
+# Pytoqlik()
+Pytoqlik() instatiates a Python and Qlik connection and allows usage of the methods below to manipulate, import/export data and effectively control Qlik applications using Python. 
+
+### Usage:
+```python
+Pytoqlik(host='ws://localhost:4848/app')
+```
+
+### Argument breakdown:
+- **host**: string. Set the host address of your Qlik Server. Default: 'ws://localhost:4848/app'.
+
+### Example:
+```python
+import pytoqlik
+p2q = pytoqlik.Pytoqlik()
+```
+
+------------------------------------------------------------------------------------------
+# toQlik()
+toQlik() is a method called on a Pytoqlik object previously created. (Using Data Load Editor!) It is used to create a new Qlik application using pandas DataFrames as data sources, or to append pandas DataFrames to an existing Qlik application by setting the appName and/or sheetName. Running toQlik() on an existing app will overwrite the scripts being run in it. 
+
+The data is loaded using Qlik's Data Load Editor, and as such, you might want to synchronize the scripted tables inside Qlik once you load your pandas DataFrame.
+
+### Usage:
+```python
+Pytoqlik().toQlik(*df, appName='PythonApp', sheetName='Dashboard', redirect=False, embedded=True, replace=True, verbose=False, width = 980, height = 800, decimal='.', separator=';')
+```
+
+### Argument breakdown:
+- **df**: pandas DataFrame. This data structure will be converted to a .csv file and added to desired Qlik application. This is a required argument, and you can pass multiple DataFrames in a single call of toQlik().
+- **appName**: string. The data will be loaded to the Qlik application with this name. If no existing applications match the name, a new one will be created. Default: 'PythonApp'.
+- **sheetName**: string. The command automatically creates a new sheet inside de application for you to work on, whose name can be set in this argument. Default: 'Dashboard'.

```

Comment:
Typo? Change to "the". 
/Python/QlikSense developer from Sweden who is curious about this project!

Reply:
Sure thing, my mistake! Fixing it asap.

And by the way, thank you for the interest. I'll be helping the project out for some time and try to bring it to life!